docs/Tutorials/Create-Private-Clique-Network.md

```diff
@@ -0,0 +1,271 @@
+
+```json
+{
+  "config":{
+    "chainId":1981,
+    "clique":{
+      "blockperiodseconds":15,
+      "epochlength":30000
+    }
+  },
+  "coinbase":"0x0000000000000000000000000000000000000000",
+  "difficulty":"0x1",
+
+"extraData":"0x0000000000000000000000000000000000000000000000000000000000000000<Node 1 Address>0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+  "gasLimit":"0xa00000",
+  "mixHash":"0x0000000000000000000000000000000000000000000000000000000000000000",
+  "nonce":"0x0",
+  "timestamp":"0x5c51a607",
+  "alloc": {
+    "<Node 1 Address>": {
+      "privateKey": "<Node 1 Private Key>",
+      "balance": "90000000000000000000000"
+    }
+  },
+  "number":"0x0",
+  "gasUsed":"0x0",
+  "parentHash":"0x0000000000000000000000000000000000000000000000000000000000000000"
+}
+```
+
+Replace:
+ 
+ * `<Node 1 Address>` with the [address for Node-1](#3-get-address-for-node-1) in `extraData` and `alloc`. 
+ * `<Node 1 Private Key>` with the private key in the `key` file in the `Node-1-data-path` directory excluding the 0x prefix. 
+
+!!! example
+    
+    ```json
+    {
+      "config":{
+        "chainId":123,
+        "clique":{
+          "blockperiodseconds":15,
+          "epochlength":30000
+        }
+      },
+      "coinbase":"0x0000000000000000000000000000000000000000",
+      "difficulty":"0x1",
+    
+    "extraData":"0x0000000000000000000000000000000000000000000000000000000000000000b9b81ee349c3807e46bc71aa2632203c5b4620340000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000",
+      "gasLimit":"0xa00000",
+      "mixHash":"0x0000000000000000000000000000000000000000000000000000000000000000",
+      "nonce":"0x0",
+      "timestamp":"0x5c51a607",
+      "alloc": {
+        "b9b81ee349c3807e46bc71aa2632203c5b462034": {
+          "privateKey": "4ff55c423e4136c1a118721aaaadf0271f823c42b7d1438c6706efe0c001f7fe",
+          "balance": "90000000000000000000000"
+        }
+      },
+      "number":"0x0",
+      "gasUsed":"0x0",
+      "parentHash":"0x0000000000000000000000000000000000000000000000000000000000000000"
+    }
+    ```
+
+!!! warning
+    Do not use the accounts in the genesis file above on mainnet or any public network except for testing.
+    The private keys are displayed so the accounts are not secure. 
+
+### 5. Delete Database Directory
+
+Delete the `database` directory created when [getting the public key for Node-1](#2-get-public-key-for-node-1).
+The node cannot be started with the Clique genesis file while the previously generated data is in the `database` directory. 
+
+### 6. Start First Node as Bootnode
+
+Start Node-1 specifying:
+
+* No arguments for the [`--bootnodes`](../Reference/Pantheon-CLI-Syntax.md#bootnodes) option because this is your bootnode
```

Comment:
if a custom genesis is used, the empty bootnodes list is already the default, setting the option is not needed.

Reply:
@jakehaugen - what are you thoughts on this?  I'm not sure whether indicating a node is a bootnode by omission is ideal (even though the option isn't strictly required now)

Reply:
We did used to need to do that but not anymore. I'm going to deal with that as a seperate PR.